examples/resnet20_cifar10.py

```diff
@@ -115,7 +115,7 @@ def resnet20():
     if args.n_test_samples is None:
         args.n_test_samples = 10000
     mlg_validate_ds = validate_ds.take(args.n_test_samples)
-    mlg_validate_ds = mlg_validate_ds.map(lambda x, y: (x, y[0]), num_parallel_calls=tf.data.AUTOTUNE)
+    mlg_validate_ds = mlg_validate_ds.map(lambda x, y: ((x,), (y[0],)), num_parallel_calls=tf.data.AUTOTUNE)
```

Comment:
This is kinda ugly. In the very common case where there's only one input and output, would it be possible for ``evaluate`` and ``evaluate_batched`` to stick the arguments  into a tuple automagically?